debezium-scripting/debezium-scripting/src/test/resources/wasm/go/router1.go

```diff
@@ -6,8 +6,8 @@ import (
 
 //export process
 func process(proxyPtr uint32) uint32 {
-	// 1
-	return debezium.SetInt(1)
+	// boolean
+	return debezium.SetBool(false)
```

Comment:
Since we removed `SetInt` from the API it's hard for the user to pass an invalid value, the only mismatch can be across boolean and strings.